crates/utils/src/utils/validation.rs

```diff
@@ -126,10 +126,10 @@ fn has_3_permitted_display_chars(name: &str) -> bool {
 }
 
 // Can't do a regex here, reverse lookarounds not supported
-pub fn is_valid_display_name(name: &str, actor_name_max_length: usize) -> LemmyResult<()> {
+pub fn is_valid_display_name(name: &str, actor_name_max_length: i32) -> LemmyResult<()> {
   let check = !name.starts_with('@')
     && !name.starts_with(FORBIDDEN_DISPLAY_CHARS)
-    && name.chars().count() <= actor_name_max_length
+    && name.chars().count() as i32 <= actor_name_max_length
```

Comment:
use `try_from` or `try_into` instead of `as`

Reply:
Will be fixed with https://github.com/LemmyNet/lemmy/pull/5688